### DISCUSSION TITLE
DOTRAND.COM - Random Fake Data Generator

### DISCUSSION THREAD

**Discussion by trinhthinh388** [heroui-inc/heroui#1297] [Showcase]
I use NextUI v2 to create this website to help people easily generate random fake data (phone numbers, emails, credit cards, etc)

Site: [https://dotrand.com](https://dotrand.com)

**Comment by PR0F1L3R1** [comment]
Awesome Dropdown selection! Can You share code of this?

**Reply by trinhthinh388** [reply]
thanks mate!
Which one do you mean mate?

**Reply by PR0F1L3R1** [reply]
![image](https://github.com/nextui-org/nextui/assets/40261445/8e7b1ce5-833e-4438-b496-2ae60a2034af)
![image](https://github.com/nextui-org/nextui/assets/40261445/16e74c7b-79c1-4a27-9c32-9ee7d6636424)

**Reply by trinhthinh388** [reply]
ah! this one I use `PopOver` and `react-window` to render the options list.
And some small stuffs like Selected Icon, etc is created from scratch :D
